From bc233dd82b86cb63a9f4b35bc300301c4d247e59 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Sun, 9 Feb 2014 00:50:01 +0100
Subject: [PATCH 10/10] MINOR: stats: report exact last session time in backend
 too

I forgot to remove one human_time() in the CSV output for the backend's
lastsess entry in previous patch, which caused the value to be reported
as "1m18s" for example instead of 78.
---
 src/dumpstats.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/dumpstats.c b/src/dumpstats.c
index d147d0f..0b7dc08 100644
--- a/src/dumpstats.c
+++ b/src/dumpstats.c
@@ -3242,7 +3242,7 @@ static int stats_dump_be_stats(struct stream_interface *si, struct proxy *px, in
 		              /* queue : current, max */
 		              "%d,%d,"
 		              /* sessions : current, max, limit, total, lastsess */
-		              "%d,%d,%d,%lld,%s,"
+		              "%d,%d,%d,%lld,%d,"
 		              /* bytes : in, out */
 		              "%lld,%lld,"
 		              /* denied: req, resp */
@@ -3268,7 +3268,7 @@ static int stats_dump_be_stats(struct stream_interface *si, struct proxy *px, in
 		              px->id,
 		              px->nbpend /* or px->totpend ? */, px->be_counters.nbpend_max,
 		              px->beconn, px->be_counters.conn_max, px->fullconn, px->be_counters.cum_conn,
-		              human_time(be_lastsession(px), 1),
+		              be_lastsession(px),
 		              px->be_counters.bytes_in, px->be_counters.bytes_out,
 		              px->be_counters.denied_req, px->be_counters.denied_resp,
 		              px->be_counters.failed_conns, px->be_counters.failed_resp,
-- 
1.8.3.2

